vendor/assets/javascripts/babel.js

```diff
@@ -50966,7 +50966,7 @@ return /******/ (function(modules) { // webpackBootstrap
 		var regexSingleEscape = /["'\\\b\f\n\r\t]/;
 
 		var regexDigit = /[0-9]/;
-		var regexWhitelist = /[ !#-&\(-\[\]-~]/;
+		var regexAllowlist = /[ !#-&\(-\[\]-~]/;
```

Comment:
Will this regress if we update babel in the future?

Reply:
Good point. This PR shouldn't touch any vendored libraries and their APIs. Those hopefully will be changed upstream and then we'll be able to change them.